cpp/src/io/csv/csv_reader.cu

```diff
@@ -53,12 +53,9 @@
 
 #include "cudf.h"
 #include "utilities/error_utils.h"
- 
-#include "rmm/rmm.h"
-
-#include "NVStrings.h"
+#include "utilities/type_dispatcher.hpp"
 
-constexpr int32_t HASH_SEED = 33;
+#include "rmm/rmm.h"
 
 using namespace std;
```

Comment:
Avoid using the `using namespace` at global scope. This almost always ends up leading to hard to track down problems.

Reply:
Looks like someone else has already removed it in the latest code.